### DISCUSSION TITLE
Mention ichthyosaurus' `sailfish-patch` in the Wiki?

### DISCUSSION THREAD

**Discussion by Olf0** [sailfishos-patches/patchmanager#293] [Ideas]
Idea: Quickly evaluate ichthyosaurus' `sailfish-patch` and if it is fine, briefly describe it and link to it an a new page in our Wiki.

See https://github.com/ichthyosaurus/sailfish-patch
`sailfish-patch` is a helper tool for developing SailfishOS patches.

**Comment by nephros** [comment]
Absolutely.

While I don't use it personally, it's a great tool.

**Reply by Olf0** [reply]
Haha, you already put it into the PM-Wiki last November: https://github.com/sailfishos-patches/patchmanager/wiki/Creating-a-Patch :smiley:
